src/datastore/team.cpp

```diff
@@ -24,10 +22,8 @@ Team::Team()
  * @param leag Team's league
  */
 Team::Team(const std::string& name, League leag)
```

Comment:
Uh why is this spelled leag instead of league

Reply:
Because `league` is a public variable : )

Reply:
Oh dang, you're right